packages/cozy-client/src/store/queries.js

```diff
@@ -141,8 +141,9 @@ const updateData = (query, newData) => {
   const toUpdate = intersection(originalIds, matchedIds)
 
   const changed = toRemove.length || toAdd.length || toUpdate.length
-
-  const updatedData = difference(union(originalIds, toAdd), toRemove)
+  // concat doesn't check duplicates (contrarily to union), which is ok as
+  // toAdd has no originalIds, and is faster.
```

Comment:
// toAdd does not contain any id present in originalIds, by construction

? I had trouble to understand at first.

Reply:
Right, I'll clarify